src/services/queries/Providers.js

```diff
@@ -3,7 +3,7 @@ import gql from 'graphql-tag';
 export const PROVIDERS = gql`
   query getListOfProviders(
     $query: String = "%%"
-    $limit: Int = 10
+    $limit: Int = 100
```

Comment:
Why increase this one? I think 10 is good

Reply:
The pagination was removed from the designer, I remember that then we talk about a limit of 100 providers, and if DID wanted to show more providers / DID have more than 100 providers (It is unlikely), then in an expansion will added a pagination or updated the limit

Reply:
Hmm OK, sound fine then